src/main/scala/io/opentargets/etl/backend/drug/Molecule.scala

```diff
@@ -87,6 +88,8 @@ object Molecule extends LazyLogging {
         when(col("parentId") === col("id"), typedLit(null)).otherwise(col("parentId"))
       )
       .drop("mol_synonyms", "synonym_type")
+      .join(probes.filter(col("drugId").isNotNull).select(col("drugId").as("chemicalProbeDrugId")).distinct(),
```

Comment:
I'd move the operations on the probes dataframe to the part of the preparation of the raw input dataframes (line 40 in `Drug.scala`)

Reply:
@ireneisdoomed just to confirm, we don't want the `chemicalProbeDrugId` column in the `molecule` dataset?